Purchases/Networking/Backend.swift

```diff
@@ -315,24 +315,24 @@ class Backend {
 
         httpClient.performGETRequest(serially: true,
                                      path: path,
-                                     headers: authHeaders) { [weak self] (statusCode, response, error) in
+                                     headers: authHeaders) { [weak self] (statusCode, maybeResponse, maybeError) in
```

Comment:
this isn't entirely related, it was just to remind myself that the response here _could_ be nil